docs/guide/troubleshooting/errors.md

```diff
@@ -1,9 +1,15 @@
-# Coming Soon
+# Common Errors
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+Getting an error running your test suite! Never fear - certainly someone before you has received the same one and can help you. Here are some common errors that might come up in your testing and solutions for how you can fix them. 
 
-Please Stay tuned for more details of this feature!
-:::
+### Incorrect handling of nonces
```

Comment:
Maybe add a note that this is usually incremented automatically by client libraries like ethers, but there might be situations where this isn't possible (like when you prepare multiple offline signed transactions and submitting them as a batch)